modules/Core/includes/endpoints/VerifyMinecraftEndpoint.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @param int $id The NamelessMC user's ID
+ * @param string $code The NamelessMC user's reset code, used to verify they own the account
+ *
+ * @return string JSON Array
+ */
+class VerifyMinecraftEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'verifyMinecraft';
+        $this->_module = 'Core';
+        $this->_description = 'Validate/Activate a NamelessMC account by confirming their reset code';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.